src/www/ui/page/AdminGroupUsers.php

```diff
@@ -137,6 +144,9 @@ function setNewPermUrl(newperm){
     }
 
     $vars['scripts'] = $scripts;
+    if(!empty($text)) {
+      $vars['scripts'] .= displayMessage($text);
```

Comment:
Won't it be be better to have something like this?
```suggestion
      $vars['message'] .= $text;
```

It will also give consistent view for error messages.